wordpress-mobile/WordPress-Android
# My Site Dashboard: Phase 1 - Quick Actions

Parent #15193

This PR includes
- UI changes according to the latest zeplin design docs (margins/paddings, font size, card border, button border)
- Rename `quick_start_toolbar` to `my_site_card_toolbar`
- Added a new layout file for the quick actions called `my_site_quick_actions_card`. This is the wrapper and includes the existing `quick_actions_block` layout file and the `my_site_card_toolbar`. Perhaps we **should** revisit names?
- Extracted the building of QuickActionsBlock to it's own builder `QuickActionsBlockBuilder` + associated tests in `QuickActionsBlockBuilderTest`.
- Updated unit tests in `MySiteViewModelTest`


Dark | Light
-- | --
<img width="250" height="500" alt="Alt desc" src="https://user-images.githubusercontent.com/506707/130860294-f9b6d733-e102-4400-95d0-2e67bd2a9a96.png"> | <img width="250" height="500" alt="Alt desc" src="https://user-images.githubusercontent.com/506707/130860301-83e6261d-93ce-468a-97f8-5748d26ebe88.png">


**To test:**

**Test Quick Actions Clicks are functional**
- Launch the app 
- Navigate to My Site -> App Settings -> Test feature configuration
- Enable the `MySiteImprovementsFeatureConfig` and tap the button to restart the app 
- Navigate to the My Site tab
- Tap on all the quick starts button to ensure they are working correctly

**Test that Pages doesn't show in quick actions**
- Navigate to My Site
- Using the site selector, select a site that you don't have edit capability for
- On the My Site tab, the quick actions card should not show the Pages action

**Test that Pages action shows blue focus icon**
- Navigate to My Site
- Using the site selector, select a site that Quick Tasks are enabled
- Tap on the Customize Your Site to launch the dialog
- Tap on Review Site Pages
- Note that the Pages quick action button shows the flashing blue focus point.
 
**Test that Pages action shows blue focus icon**
- Navigate to My Site
- Using the site selector, select a site that Quick Tasks are enabled
- Tap on the Grow your audience to launch the dialog
- Tap on Check your stats
- Note that the Stats quick action button shows the flashing blue focus point.

**Note**
Make toolbar "More" invisible - this part will be handled separately.

## Regression Notes
1. Potential unintended areas of impact
Quick Actions navigation does not work

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
`MySiteViewModelTest` and `QuickActionsBlockBuilderTest`

PR submission checklist:
